src/common/components/Header/components/AdminButtonPanel/AdminButtonPanel.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Button from 'src/client/components/Common/Button';
+import styles from './style.scss';
+import {HOME_PAGE, TOURNAMENT_PAGE, TASK_TRAIN_PAGE} from 'common/constants/activePageNames';
+
+class AdminButtonPanel extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+
+  handleClick = () => {
+  };
+
+  buttonRouteMap = {
+    newTournament: {
+      path: '',
+      buttonName: 'New Tournament'
+    },
+    editTournament: {
+      path: '',
+      buttonName: 'Edit Tournament'
+    },
+    newTask: {
+      path: '',
+      buttonName: 'Add New Task'
+    },
+    editTask: {
+      path: '',
+      buttonName: 'Edit Task'
+    }
+  };
+
+  pageNameComponentMap = {
+    [HOME_PAGE]: <Button type='submit'
+      mod='success'
+      onClick={this.handleClick(this.buttonRouteMap.newTournament.path)}>
+      {this.buttonRouteMap.newTournament.buttonName}
+    </Button>,
+    [TOURNAMENT_PAGE]: [
+      <Button type='submit'
+        key={'editTournamentButton'}
+        onClick={this.handleClick(this.buttonRouteMap.editTournament.path)}>
```

Comment:
Is it okay to stay on this stage of code - blank handler functionality (redirect)?) It will be in next PR?

Reply:
As I may understand this is just the Home Page)

Reply:
@SergeyKovalchuk Yes, the rest  of the logic  will be done  later when  some pages(Edit task page,  edit tournament page) will be implemented.